src/Clock.elm

```diff
@@ -0,0 +1,103 @@
+module Main exposing (Model, Msg(..), init, main, subscriptions, update, view)

```

Comment:
You shouldn't need to expose `Msg`'s constructors & `main`:
Your `main` is not really useful outside of this module.
If an external module imports your `Msg`'s constructor it means it has direct access to your internal `update`, which is generally a bad idea.